tools/wpt/testfiles.py

```diff
@@ -278,7 +322,7 @@ def affected_by_interfaces(file_contents):
                 continue
 
             with open(test_full_path, "rb") as fh:
-                file_contents = fh.read()
+                file_contents = fh.read()  # type: Union[bytes, Text]
```

Comment:
The file is opened with "rb", so I'd hope this will always be bytes.

Reply:
It'll always be `bytes` here, but `file_contents` has the `Union` type because we then decode it. I guess we can add a new variable here.